full-service/src/json_rpc/json_rpc_request.rs

```diff
@@ -133,6 +133,10 @@ pub enum JsonCommandRequest {
     get_account_status {
         account_id: String,
     },
+    get_address_for_account {
```

Comment:
Is it returning the public address? b58 encoded?

Reply:
The full address object. Updated to specify the type instead of generic json value